content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -104,7 +104,8 @@ different Kubernetes components.
 | `NodeLease` | `false` | Alpha | 1.12 | |
 | `PersistentLocalVolumes` | `false` | Alpha | 1.7 | 1.9 |
 | `PersistentLocalVolumes` | `true` | Beta | 1.10 | |
-| `PodPriority` | `false` | Alpha | 1.8 | |
+| `PodPriority` | `false` | Alpha | 1.8 | 1.10 |
+| `PodPriority` | `true` | Beta | 1.11 | |
```

Comment:
PodPriority graduated to beta in 1.11 and was in beta till 1.13. It graduated to GA in 1.14.